docs/source/index.rst

```diff
@@ -63,12 +63,16 @@ The value(s) defined in the pytest configuration file may be accessed as a list
 by test code via the ``pytestconfig`` fixture which must be passed in as an
 argument to the test method or function that will use the value.
 
-Example of accessing configuration values within tests::
+Example of accessing configuration values within test code itself::
 
   def test_important_thing(pytestconfig):
       setup_cfg_inputs_root = pytestconfig.getini('inputs_root')[0]
       assert setup_cfg_inputs_root == 'my_data_repo'
+      
+From within a fixture or a test class the configuration values must be accessed using a slightly different approach.::
 
+Example:
```

Comment:
Is this extra "Example:" text necessary? If so, why not move it to before `::` so block quote would render correctly?